sdk/resources/Microsoft.Azure.Management.Resource/src/Properties/AssemblyInfo.cs

```diff
@@ -8,7 +8,7 @@
 [assembly: AssemblyDescription("Provides Microsoft Azure resource management operations including Resource Groups.")]
 
 [assembly: AssemblyVersion("3.0.0.0")]
-[assembly: AssemblyFileVersion("3.16.2.0")]
```

Comment:
Here the `AssemblyFileVersion` should be `3.17.1.0` and the `AssembluFileVersion` should be `3.17.1.0` as well, aligned with the version in `csproj`.